modules/flink-ext/README.txt

```diff
@@ -20,7 +20,7 @@ interested in):
         ...
         <dependency>
             <groupId>org.apache.ignite</groupId>
-            <artifactId>ignite-flink</artifactId>
+            <artifactId>ignite-flink-ext</artifactId>
             <version>${ignite.version}</version>
```

Comment:
Should we replace the _ignite.version_ parameter with _ignite-flink-ext_ one as long as the version of an extension is different from the Ignite one.

Reply:
@dmagda thank you for reviewing the changes, I have updated the PR.

Reply:
@samaitra, sorry for a belated check. All looks good to me, please go ahead and merge the changes to the master.

Reply:
@dmagda thank you, I have merged the changes.